sql/api/src/main/scala/org/apache/spark/sql/MergeIntoWriter.scala

```diff
@@ -250,9 +250,7 @@ case class WhenMatched[T] private[sql] (
  * @tparam T
  *   The type of data in the MergeIntoWriter.
  */
-case class WhenNotMatched[T] private[sql] (
-    mergeIntoWriter: MergeIntoWriter[T],
-    condition: Option[Column]) {
+case class WhenNotMatched[T](mergeIntoWriter: MergeIntoWriter[T], condition: Option[Column]) {
```

Comment:
Consider keeping the original wrapped format.

Reply:
I meant the original code was wrapped so you could keep the extra vertical lines of code minus the `private[sql]`.